docs/docs/en/guide/installation/pseudo-cluster.md

```diff
@@ -83,6 +83,8 @@ each machine. You could find this file in the path `bin/env/install_env.sh` and
 # ---------------------------------------------------------
 # Due to the master, worker, and API server being deployed on a single node, the IP of the server is the machine IP or localhost
 ips="localhost"
+# Modify Configurationexport export sshPort="22"
```

Comment:
We could remove this comment as we have already had it above.

Reply:
Agree with that